source/reference/access-mdb-shell-help.txt

```diff
@@ -412,3 +430,11 @@ certain database commands and obtain information on your deployment:
      - Display a list of all roles, both user-defined and built-in, for
        the current database.
 
+   * - ``show tables``
+     - Print a list of collections in the current database. See ``show
```

Comment:
[suggestion]

Consider updating this to "Display" to be more consistent with usage on the rest of the page.